src/Confluent.Kafka/WatermarkOffsets.cs

```diff
@@ -33,7 +33,7 @@ public class WatermarkOffsets
         ///     The offset of the earlist message in the topic/partition.
         /// </param>
         /// <param name="high">
-        ///     The offset of the last committed message in the topic/partition.
+        ///     The offset of the last stored message in the topic/partition.
```

Comment:
yeah, I think so too but someone was telling me they thought stored was better because of possible confusion of the word committed with committing offsets. I'll change it back to committed for now.